typescript-light-bulb/lightbulb.ts

```diff
@@ -0,0 +1,15 @@
+const $lightOnContainer = document.querySelector('.light-container');
+const $lightOffContainer = document.querySelector('.light-off-container');
+
+if (!$lightOnContainer || !$lightOffContainer)
+  throw new Error('Light containers not found');
+
+$lightOnContainer.addEventListener('click', () => {
+  $lightOnContainer.classList.add('hidden');
+  $lightOffContainer.classList.remove('hidden');
+});
+
+$lightOffContainer.addEventListener('click', () => {
+  $lightOffContainer.classList.add('hidden');
+  $lightOnContainer.classList.remove('hidden');
+});
```

Comment:
Alternative solution carrying on from my last comment. Now since we'd only have 1 container and light, I can do this

```
let isOn = false;

const $ceiling = document.querySelector('.ceiling');
const $bulb = document.querySelector('.bulb');

if (!$ceiling) throw new Error('$ceiling does not exist');
if (!$bulb) throw new Error('$bulb does not exist');

$bulb.addEventListener('click', (event: Event) => {
  isOn = !isOn;
  if (isOn) {
    $ceiling.className = 'ceiling is-lit';
    $bulb.className = 'bulb is-on';
  } else {
    $ceiling.className = 'ceiling';
    $bulb.className = 'bulb';
  }
});
```